### PR TITLE
feat: Add Sample for `PasswordVault`-API

### PR DESCRIPTION
**Pull request by TopperDEL**
GitHub Issue (If applicable): #495

## PR Type

What kind of change does this PR introduce?

- Sample

## What is the current behavior?

No sample for `PasswordVault`.

## What is the new behavior?

A sample for `PasswordVault`.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [X] Tested on Android.
- [X] Tested on UWP.
- [X] Tested in both **Light** and **Dark** themes.
- [X] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	  xmlns:macos="http://uno.ui/macos"
+	  xmlns:skia="http://uno.ui/skia"
+	  xmlns:wasm="http://uno.ui/wasm"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  mc:Ignorable="d android ios macos skia wasm"
+	  x:Name="page">
+
+    <Grid Background="{StaticResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.FluentTemplate>
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml:20]: We should have a `DesignAgnosticTemplate`, as we do in the [Uno.Toolkit Sample App](https://github.com/unoplatform/uno.toolkit.ui/blob/17369bd6f9b16288c74e5e28716e1e69c28e099e/samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Controls/SamplePageLayout.Properties.cs#L153-L167)

Not a blocker for this PR but something that can be updated to use in the future

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.Features, "PasswordVault", Description = "The PasswordVault-API enables storing and retrieving of secret data in Uno applications.", DocumentationLink = "https://platform.uno/docs/articles/features/PasswordVault.html")]
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml.cs:7]: This should probably use the official WinRT API docs for the DocumentationLink (https://learn.microsoft.com/en-us/uwp/api/windows.security.credentials.passwordvault) as well as the official Description: "Represents a Credential Locker of credentials. Lockers are specific to a user."

[the remark this answers is not in the document]
Comment (TopperDEL) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml.cs:7]: I've added a SampleConditionalAttribute - but I have the feeling that it is wrong now. #tired

Comment (TopperDEL) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml.cs]: Fixed both!

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml (hunk 1 above)
Comment (MartinZikmund) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml:20]: No need , reported as separate issue

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@TopperDEL The `SampleConditionalAttribute` is a bit confusing indeed. I have added the needed adjustment and with that it is now good to go!

**Comment by TopperDEL** [comment]
Awesome, thanks!

**Review by kazo0 — APPROVED** [review]

**Pull request merged**